.goreleaser.yml

```
@@ -48,6 +48,16 @@ archives:
 release:
   draft: true
 
+
+git:
+  ignore_tags:
+    - "{{ if not .IsNightly }}nightly{{ end }}"
+
+nightly:
+  publish_release: true
+  keep_single_release: true
+  version_template: "{{ incminor .Version }}-nightly"
```

Just standardizing Go template a bit:

```suggestion
git:
  ignore_tags:
    - "{{if not .IsNightly}}nightly{{end}}"

nightly:
  publish_release: true
  keep_single_release: true
  version_template: "{{incminor .Version}}-nightly"
```